packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -431,6 +450,29 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
       );
     }
   }
+
+  Future<void> scrollVerticallyToData(V data) async {
+    await verticalScrollController.animateTo(
+      // Subtract [2 * rowHeightWithPadding] to give the target scroll event top padding.
+      topYForData(data) - 2 * rowHeightWithPadding,
```

Comment:
you should take the min of this and zero to avoid a bounce when scrolling to the top. We're also scrolling too far when the item is close to the bottom.

Reply:
hmm. I'm seeing some bounce animations (top of the scrollable flashes blue if using mtv not cupertino style) due to similar bugs in the inspector.

Reply:
added a clamp operation